src/main/java/core/basesyntax/ColorBall.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class ColorBall {
+    public enum Color {
+        RED,
+        GREEN,
+        BLUE,
+        YELLOW,
+        ORANGE
+    }
+}
```

Comment:
```suggestion
    public enum Color {
        RED,
        GREEN,
        BLUE,
        YELLOW,
        ORANGE
    }
```

Reply:
not fixed, you don't enclosing class, just use enum